app/views/microposts/update.js.erb

```diff
@@ -0,0 +1,3 @@
+alert("Hurray Blog has been successfully edited....");
+$("#blog-edit").html("<%=@micropost.content%>");
```

Comment:
@HrisikeshSinghaChowdhury how is it working?  how will it find the specific blog row for which you are updating the content?

Reply:
it's working only first time